App.Loans/Services/ValidateServices.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using App.Loans.Models;
+
+namespace App.Loans
+{
+    public class ValidateServices : IValidateServices
+    {
+        public void ValidateLoan(Loan loan)
+        {
+            if (loan == null)
+                throw new Exception("Loan doesn't exist");
```

Comment:
This is not the responsibility of Validator. Null checking is the responsibility of caller method